bump-bot/bump_bot.py

```diff
@@ -35,8 +35,8 @@ async def reaction_changed(payload: discord.RawReactionActionEvent, added: bool)
 	message = await message_cache.get_message(payload.message_id, channel)	
 	if message:
 
-		# reaction_removed events have no "member", so we grab them via the user_id
-		member = discord.utils.get(client.get_all_members(), id=(payload.member.id if payload.member is not None else payload.user_id))
+		# reaction_removed events do not have a member included while reaction_added ones do.
```

Comment:
This comment is a bit confusing now that we don't actually use the member in either case. If this just works as is without using payload.member, I'd just remove the comment